api/v1/model_types.go

```diff
@@ -20,12 +20,12 @@ type ModelSpec struct {
 	// Resources are the compute resources required by the container.
 	Resources *Resources `json:"resources,omitempty"`
 
-	// BaseModel should be set in order to mount another model to be
+	// Model should be set in order to mount another model to be
 	// used for transfer learning.
-	BaseModel *ObjectRef `json:"baseModel,omitempty"`
+	Model *ObjectRef `json:"model,omitempty"`
```

Comment:
I think this rename makes sense.

However, I am not a big fan of renames like these since those result in a lot of downstream changes to the examples and docs as well. The value we get out of these is medium but the amount of effort to validate the change doesn't break things is significant.